layouts/shortcodes/pages/get-involved/testimonials.html

```diff
@@ -0,0 +1,49 @@
+{{ $lang := .Page.Lang }}
+{{$langBase:=index .Site.Data $lang}}
+
+{{ $scratch := newScratch }}
+{{ range $langBase.testimonials.categories }}
+    {{ if .title }}
+        {{ $scratch.Set "category_title" (.title | urlize) }}
+        {{ $scratch.SetInMap "categories" (.title | urlize) . }}
+    {{ end }}
+    {{ range $index, $testimonial := .testimonials }}
+        {{ $category := dict "category" ($scratch.Get "category_title") }}
+        {{ $testimonial = merge . ($category) }}
+        {{ $scratch.SetInMap "testimonials" (.name | urlize) $testimonial }}
+    {{ end }}
+{{ end }}
+
+{{ $testimonials := $scratch.GetSortedMapValues "testimonials" }}
+
+{{ if $testimonials }}
+<div class="get-involved-testimonials testimonials-alternate row">
+    
+        <h2 class="big-text-secondary text-center margin-bottom-40">{{ i18n "home-testimonials" }}</h2>
+        
+            <ul class="solstice-slider owl-carousel owl-theme list-inline" data-slider-lg-count="1" data-slider-md-count="1" data-slider-sm-count="1">
```

Comment:
Owl injects stuff between the items and the parent container, so the list items aren't semantically correct here. You should just use divs

Reply:
I replaced the <ul>/<li> tags with <div> tags.